examples/cfd/aero_graph_net/README.md

```diff
@@ -0,0 +1,210 @@
+4. [Training](#model-training)
+    1. [Ahmed Body](#ahmed-body-training)
+    2. [DrivAerNet](#drivaer-training)
+5. [Inference](#inference)
+
+## Problem overview
+
+To goal is to develop an AI surrogate model that can use simulation data to learn the
+external aerodynamic flow over parameterized car body shape. The trained model can be used
+to predict the change in drag coefficient,and surface pressure and wall shear stresses due
+to changes in the car geometry. This is a stepping stone to applying similar approaches
+to other application areas such as aerodynamic analysis of aircraft wings, more complex
+real car geometries, and so on.
+
+## Datasets
+
+AeroGraphNet currently supports two datasets: [Ahmed Body](#ahmed-body) and
+[DrivAerNet](#drivaernet).
+
+### Ahmed Body
+
+Industry-standard Ahmed-body geometries are characterized by six design parameters:
+length, width, height, ground clearance, slant angle, and fillet radius. Refer
+to the [[2, 3](#references)] for details on Ahmed
+body geometry. In addition to these design parameters, we include the inlet velocity to
+address a wide variation in Reynolds number. We identify the design points using the
+Latin hypercube sampling scheme for space filling design of experiments and generate
+around 500 design points.
+
+The aerodynamic simulations were performed using the GPU-accelerated OpenFOAM solver
+for steady-state analysis, applying the SST K-omega turbulence model. These simulations
+consist of 7.2 million mesh points on average, but we use the surface mesh as the input
+to training which is roughly around 70k mesh nodes.
+
+To request access to the full dataset, please reach out to the
+[NVIDIA Modulus team](modulus-team@nvidia.com).
+
+### DrivAerNet
+
+DrivAerNet [[5](#references)] is a larger dataset which contains around 4000 high-quality
+car meshes, coefficients and flow information.
+The dataset can be downloaded by following the instructions on the [DrivAerNet GitHub](https://github.com/Mohamedelrefaie/DrivAerNet)
+Please see the corresponding [paper](#references) for more details.
+
+## Model overview and architecture
+
+The AeroGraphNet model is based on the MeshGraphNet [[1](#references)] architecture
+which is instrumental for learning from mesh-based data using GNNs.
+Depending on the dataset, the model takes different inputs:
+
+### Ahmed Body dataset
+
+- Ahmed body surface mesh
+- Reynolds number
+- Geometry parameters (optional, including length, width, height, ground clearance,
+slant angle, and fillet radius)
+- surface normals (optional)
+
+Output of the model are:
+
+- Surface pressure
+- Wall shear stresses
+- Drag coefficient - optional, computed using pressure and shear stress outputs.
+
+![Comparison between the AeroGraphNet prediction and the
+ground truth for surface pressure, wall shear stresses, and the drag coefficient for one
+of the samples from the test dataset.](../../../docs/img/ahmed_body_results.png)
```

Comment:
Would be very cool if we could add some images like this for the drivernet dataset too :)

Reply:
Done - added an image, updated the README.

Reply:
Thanks! Looks great!